### DISCUSSION TITLE
Problem: classFactory() NameError: name 'jit' is not defined.

### DISCUSSION THREAD

**Discussion by Bugas1234** [UMEP-dev/UMEP#512] [General]
Dear UMEP team,

With the last update the Daily Shadow Pattern tool was moved to UMEP for processing, but I have not been able to download it properly.
It gives me the following error : It was not possible to load the module 'processing_umep' due to an error calling your method classFactory()
NameError: name 'jit' is not defined.
Do you know how to solve this?
I have the most recent Qgis and Python versions installed.

-->Update: 
Fredrik Lindberg
A quick fix for you is to install the numba package via the osgeo installer.

-->Update:
It worked! I had also to install javax64

Many thanks to UMEP team.

**Comment by biglimp** [comment]
Try to uninstall the tool and reinstall it again and let us know what the result is.

**Reply by Bugas1234** [reply]
Hello, I had tried that before and did not work. 
With the help of Fredrik Lindberg it was possible to get to the solution which is install the numba package via the osgeo installer.

**Comment by biglimp** [comment]
Sorry, this is Fredrik (just another username). 
So I am confused. Where you able to start UMEP for processing when you installed numba?

**Reply by Bugas1234** [reply]
Ah, sorry!

Yes. As you had suggested by installing numba through osgeo installer it was possible to get it to work. After it, I had also to install javax64 since I only had x32.

Then, in the toolbar I went to processing --> processing tools --> UMEP

**Reply by biglimp** [reply]
Great. We will work on a easy solution to get the correct python packages for the new version. Should be ready soon.

**Reply by Bugas1234** [reply]
Once again, many thanks for the fix! Great job! :)
